docs/core/packages.md

```diff
@@ -40,76 +40,45 @@ The following is a list of the key NuGet packages for .NET Core:
 - [System.Linq](https://www.nuget.org/packages/System.Linq) - A set of types for querying objects, including Enumerable and [ILookup&lt;TKey, TElement&gt;](http://docs.microsoft.com/dotnet/core/api/System.Linq.ILookup-2).
 - [System.Reflection](https://www.nuget.org/packages/System.Reflection) - A set of types for loading, inspecting and activating types, including [Assembly](http://docs.microsoft.com/dotnet/core/api/System.Reflection.Assembly), [TypeInfo](http://docs.microsoft.com/dotnet/core/api/System.Reflection.TypeInfo) and [MethodInfo](http://docs.microsoft.com/dotnet/core/api/System.Reflection.MethodInfo).
 
-Packages are referenced in project.json. In the example below, the [System.Runtime](https://www.nuget.org/packages/System.Runtime/) package is referenced. 
-
-```json
-{
-  "dependencies": {
-    "System.Runtime": "4.1.0"
-  },
-  "frameworks": {
-    "netstandard1.6": {}
-  }
-}
+In most cases, you do not need to reference the lower-level .NET Core packages directly since you'll end up with too many packages to manage. However, you can reference individual packages in your MSBuild project file. In the example below, the [System.Runtime](https://www.nuget.org/packages/System.Runtime/) package is referenced. 
```

Comment:
I'd try to state this more generally. We are going to have metapackages for the Azure SDK soon. One could see a future where there are other metapackages for other workloads.

On the other hand, you may only need one package from a workload.

I'd suggest something like:

> You can include packages either individually, or as a metapackage that includes sets of packages often used together. It is usually easier to include metapackages, as described later. However, when you need a single package, you include it as in the example below, with the [System.Runtime](http://nuget.org/packages/System.Runtime) package.

Feel free to edit at will. 😄